### PR TITLE
WIP - Off Chain Voting

### PR DESCRIPTION
**Pull request by bcstryker**
ready for merge - link to pages in progress is `[deployment_url]/v2/dill/vote` and `[deployment_url]/v2/dill/vote/sidechains`

### CODE REVIEW

File: public/locales/en/common.json [hunk 1]
```diff
@@ -373,8 +373,19 @@
       "upcomingDistributionDate": "Upcoming distribution date",
       "upcomingDistributionValue": "Upcoming distribution value",
       "usdValue": "USD Value",
-      "weeklyDillAmount": "Weekly DILL 🔒",
-      "weeklyPickleAmount": "Weekly PICKLEs distributed"
```
Comment (janklimo) [inline public/locales/en/common.json:377]: @bcstryker why was this removed? It breaks the dill chart labels/tooltips.

Comment (bcstryker) [reply public/locales/en/common.json:377]: wasn't supposed to be, my bad, will re-add if you haven't already

Comment (bcstryker) [reply public/locales/en/common.json:377]: weird, on checking master, this change is not present

Comment (bcstryker) [reply public/locales/en/common.json:377]: replaced and merged, my bad 😬

Comment (janklimo) [reply public/locales/en/common.json:377]: Awesome, thank you!

Comment (bcstryker) [reply public/locales/en/common.json:377]: closing this
